tests/quantization/bnb/test_4bit.py

```diff
@@ -96,6 +96,10 @@ class Base4bitTests(unittest.TestCase):
     num_inference_steps = 10
     seed = 0
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Where is this method called?

Reply:
The `@classmethod` method `setUpClass()` is automatically called once per test class, before any test methods run.

Reply:
setUp will be called before each test module but setUpClass will be called only once before all test modules in a test class. For `torch.use_deterministic_algorithms(True)`, we only need called once so it's better in the setUpClass.